### PR TITLE
Add healing links to Sacred Ground predicated on Divine Font type

### PR DESCRIPTION
**Pull request by simonward**

### CODE REVIEW

File: packs/feats/sacred-ground.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
...
+            "value": "<p><strong>Frequency</strong> once per 10 minutes</p>\n<hr />\n<p>You pray continuously for 1 minute to call a subtle shadow of your deity's realm over a @Template[type:burst|distance:30] centered on you. It lasts for 10 minutes, and a creature that remains in the area for the entire 10 minutes regains Hit Points equal to your level. If you have a @Damage[(@actor.level)[vitality,healing]]{healing font}, this activity has the healing and vitality traits and heals living creatures. If you have a @Damage[(@actor.level)[void,healing]]{harmful font}, this activity has the healing and void traits and heals undead creatures (or other creatures with void healing). Clerics with Versatile Font can choose either or both. It can't damage creatures in any case.</p>"
```
Comment (simonward) [inline packs/feats/sacred-ground.json:14]: Open to suggestion on where to place these links.

Comment (TikaelSol) [reply packs/feats/sacred-ground.json:14]: Seems like a good place for a description override, actually. Do we have a roll option for which font you have? Or maybe not?

Comment (simonward) [reply packs/feats/sacred-ground.json:14]: Looks like only a roll option for having the feature: `feature:divine-font`.
We could add a roll option for this, given that it's in the deity's structured data?

Comment (TikaelSol) [reply packs/feats/sacred-ground.json:14]: Some deities let you choose, so we would have to have the choice.

**Comment by simonward** [comment]
Screen Shot 2024-01-30 at 10 42 44 PM

**Comment by simonward** [comment]
Screen Shot 2024-01-31 at 2 02 00 PM

**Pull request merged**